### PR TITLE
chore: add `is-a.dev`

### PR DESCRIPTION
**Pull request by wdhdev**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

is-a.dev is a free subdomain service for developers. We offer subdomains on the domain `is-a.dev` to all developers for their personal websites and portfolios.

Organization Website: 

https://www.is-a.dev
https://github.com/is-a-dev/register

Reason for PSL Inclusion
====

Original PR: #1327

We are seeking to get our domain `is-a.dev` added to the Public Suffix List since it is a public domain service available to everyone.

I am an administrator of the is-a.dev project.

One of the main reasons we would like to be included on this list is **cookie separation**, cookie separation will allow all cookies across our users subdomains to remain separate and to prevent potential security risks between subdomains.

Another reason for why we should be added is, users are unable to use services such as [Vercel](https://vercel.com) and [Netlify](https://netlify.com) due to those services requiring root domain verification even when using a subdomain on their service. This is because the services use the Public Suffix List (*as far as I'm aware*) and do not recognise our domain as a public suffix.

This project is very similar in what we offer to services such as https://eu.org and https://js.org which are already on the PSL.

Number of users this request is being made to serve:

Currently over 2,660 individuals, with the count growing everyday.

DNS Verification via dig
=======

```
dig +short TXT _psl.is-a.dev
"https://github.com/publicsuffix/list/pull/1949"
```

Results of Syntax Checker (`make test`)
=========

All tests passed.

---

Please note, the submitter email is the service owner's email address, not mine. I am using that email address as he is the one to contact if there is any issues.

### CODE REVIEW

**Comment by orxngc** [comment]
Whoever approves this PR is extremely based.

**Comment by YouFoundAlpha** [comment]
Let's hope it gets approved

**Comment by TerraPlayz** [comment]
REALLY GOOD SERVICE. i hope it gets approved

**Comment by CuteDog5695** [comment]
Hope this gets approved

**Comment by SX-9** [comment]
approve please

**Comment by Glqch0** [comment]
He listed every possible reason it SHOULD be added, please approve

**Comment by SukiPlayz** [comment]
approve this service please

**Comment by simon-friedberger** [comment]
Can you please contact Netlify and Vercel? Or provide a link to their docs?
We need to keep track of how people use the PSL and make sure it's appropriate.

**Comment by simon-friedberger** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [x] is-a.dev expires 2031-10-03
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [x] Sorting
* [ ] Reasoning/Organization description

**Comment by simon-friedberger** [comment]
> This project is very similar in what we offer to services such as https://eu.org/ and https://js.org/ which are already on the PSL.

If these are all "you". Why are they not under the same organization?

**Comment by TerraPlayz** [comment]
> > This project is very similar in what we offer to services such as https://eu.org/ and https://js.org/ which are already on the PSL.
> 
> If these are all "you". Why are they not under the same organization?

They are "Similar" to what subdomains we provide. We do not come from the name owner nor the same orginization.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -13790,6 +13790,10 @@ iopsys.se
 // Submitted by Matthew Hardeman <mhardeman@ipifony.com>
 ipifony.net
 
+// is-a.dev : https://www.is-a.dev
+// Submitted by William Harrison <phenax5@gmail.com>
```
[the remark this answers is not in the document]
Comment (wdhdev) [reply public_suffix_list.dat]: I've updated the email.

**Comment by wdhdev** [comment]
> Can you please contact Netlify and Vercel? Or provide a link to their docs? We need to keep track of how people use the PSL and make sure it's appropriate.

Vercel checks the *apex* domain even if you are using a subdomain and no one else is using the root domain. If is-a.dev was listed on the PSL it will fix this as it will recognise `.is-a.dev` subdomains as individual domains instead of just subdomains. (ref: https://vercel.com/docs/projects/domains/troubleshooting#domain-ownership-errors)

With Netlify, I cannot find a doc page however, I can tell you that it does the same as Vercel regarding domain ownership.

**Comment by simon-friedberger** [comment]
Just as a note, comments of the form "+1" or "approve this" really don't add any value. ;)

**Review by simon-friedberger — APPROVED** [review]

**Comment by wdhdev** [comment]
> Just as a note, comments of the form "+1" or "approve this" really don't add any value. ;)

Yeah I know. I didn't ask people to do that if that's what you're thinking. People just did that after I announced we applied for the PSL.

**Comment by Gen1x-ALT** [comment]
approve this

**Comment by dnsguru** [comment]
piling on comments is actually slowing approval / merging as it all gets reviewed holistically, and may start the review over, so unless there is interest in delay, please stop 'nudging'

**Comment by wdhdev** [comment]
@dnsguru Can you lock the conversation? (Once again, I'm not asking people to do this.)

**Pull request merged**